plan/config.go

```diff
@@ -12,41 +14,73 @@ const defaultCallTimeout = 5
 // ReadConfigFromEnviron creates a Config by looking for CROSSDOCK_ environment vars
 func ReadConfigFromEnviron() *Config {
 	const (
-		callTimeoutKey = "CROSSDOCK_CALL_TIMEOUT"
-		clientsKey     = "CROSSDOCK_CLIENTS"
-		axisKeyPrefix  = "CROSSDOCK_AXIS_"
+		callTimeoutKey    = "CROSSDOCK_CALL_TIMEOUT"
+		waitKey           = "CROSSDOCK_WAIT_FOR"
+		axisKeyPrefix     = "CROSSDOCK_AXIS_"
+		behaviorKeyPrefix = "CROSSDOCK_BEHAVIOR_"
 	)
 	callTimeout, _ := strconv.Atoi(os.Getenv(callTimeoutKey))
 	if callTimeout == 0 {
 		callTimeout = defaultCallTimeout
 	}
-	clients := trimCollection(strings.Split(os.Getenv(clientsKey), ","))
-	var axes []Axis
+	fmt.Println(os.Getenv(waitKey))
+	waiters := trimCollection(strings.Split(os.Getenv(waitKey), ","))
+	axes := make(map[string]Axis)
+	behaviors := make(map[string]Behavior)
 	for _, e := range os.Environ() {
-		if !strings.HasPrefix(e, axisKeyPrefix) {
-			continue
-		}
-		d := strings.TrimPrefix(e, axisKeyPrefix)
+		if strings.HasPrefix(e, axisKeyPrefix) {
```

Comment:
+1. Move all this repetitive string transformation logic into a separate function with a descriptive name and documentation to give a better idea of what the inputs and outputs are.